moped-editor/src/styles/mapboxDrawStylesOverrides.js

```diff
@@ -0,0 +1,243 @@
+import theme from "src/theme/index";
+import { mapStyles } from "src/utils/mapHelpers";
+
+// Default Mapbox GL Draw styles with overrides from MUI theme
+// See https://github.com/mapbox/mapbox-gl-draw/blob/main/src/lib/theme.js
+// See https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/
+const mapboxDrawStylesOverrides = [
+  {
+    id: "gl-draw-polygon-fill-inactive",
+    type: "fill",
+    filter: [
+      "all",
+      ["==", "active", "false"],
+      ["==", "$type", "Polygon"],
+      ["!=", "mode", "static"],
+    ],
+    paint: {
+      "fill-color": "#3bb2d0",
+      "fill-outline-color": "#3bb2d0",
```

Comment:
Should these hex values import from the theme file?
https://github.com/cityofaustin/atd-moped/blob/main/moped-editor/src/theme/index.js#L26-L29

Seems like we have map specific theming ready to go.

Reply:
Good call! Just pushed this change - thanks for reminding me about that part of the theme. 🙏